### ISSUE TITLE
Enable tools in core services

### ISSUE THREAD

**Issue by RobPasMue** [ansys/pyansys-geometry#1579]
Core services seems to have catched up with the various tools. Let's activate them and verify that the tests are passing.

**Comment by jonahrb** [comment]
@RobPasMue is this considered complete now?

**Comment by RobPasMue** [comment]
Some tests are still being skipped and require to be fixed before we can consider closing it. Search for "skip_if_core_service" in the tests folder. I am hitting the following modules:

test_geometry_commands
test_prepare_tools
test_repair_tools
test_measurement_tools

I believe all the skips in those modules should be solved before we can officially say that we support the tools (not only repair but also prepare and measurement)

**Comment by jonahrb** [comment]
@RobPasMue @jacobrkerstetter i believe Jacob's PR will close this issue - bringing measurement tools?

**Comment by jonahrb** [comment]
The only thing remaining after that is find/fix small faces - the behavior is different between DMS and CoreService, but that can still close this issue and open a separate one

**Comment by RobPasMue** [comment]
Yes that's the only remaining one. Is the behavior different in the sense that we get different results? Or is the API not implemented?

**Comment by RobPasMue** [comment]
We might have to adapt the tests in case it's just different results so that they accommodate both backends. I can understand that the results might not be exactly the same.
